data/mods/ssb/moves.ts

```diff
@@ -1082,6 +1082,37 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		target: "normal",
 		type: "Ice",
 	},
+	// Celine
+	statusguard: {
+		accuracy: 100,
+		basePower: 0,
+		category: "Status",
+		desc: "Protects from physical moves. If hit by physical move, opponent is either badly poisoned, burned, or paralyzed at random and is forced out. Special attacks and status moves go through this protect.",
+		shortDesc: "If opponent uses physical attack, then gets status and phazed. User is protected.",
+		name: "Status Guard",
+		pp: 10,
+		priority: 4,
+		flags: {protect: 1},
+		onTryMovePriority: 100,  // don't understand this and next line
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
```

Comment:
these are required to make animations perform properly and fail correctly for the custom moves